tests/operations.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import {
+  createProfile,
+  deleteProfile,
+  randomProfileName,
+} from "./helpers/profile";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+
+let instance = randomInstanceName();
+let profile = randomProfileName();
+
+test.beforeAll(async ({ browser, browserName }) => {
```

Comment:
`beforeAll` and `afterAll` are not necessary in this test suite since there is only one test. Probably clearer to have these setup/tear-down steps inside the test, easier to read imo.